apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -337,6 +337,8 @@ export class Utilities {
     }
 
     const environment: IEnvironment = Utilities._createEnvironmentForRushCommand(options.initCwd);
+    // load the local node_modules/.bin directory into the PATH
+    environment.PATH = `${path.resolve(options.workingDirectory, 'node_modules', '.bin')}:${environment.path}`;
```

Comment:
Can you use a consistent case for `environment.PATH`?

Reply:
Whoops, sorry for the slop. Also switched to `environment['PATH']` while I was at it since it looked like that's how environment variable access was written elsewhere.

Reply:
Nevermind, looks like your tslint rules don't allow access through `environment['PATH']` so I just switched back to `environment.PATH`